SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialReplicationGraph.h

```diff
@@ -31,5 +31,5 @@ class SPATIALGDK_API USpatialReplicationGraph : public UReplicationGraph
 	virtual void PostReplicateActors(UNetReplicationGraphConnection* ConnectionManager) override;
 	//~ End UReplicationGraph Interface
 
-	TUniquePtr<FSpatialLoadBalancingHandler> LoadBalancingHandler;
+	FSpatialLoadBalancingHandler* LoadBalancingHandler;
```

Comment:
How about we use a TWeakPtr<FSpatialLoadBalancingHandler> here...

Reply:
addressed below